rapids-cmake/cuda/patch_toolkit.cmake

```diff
@@ -32,6 +32,7 @@ of cublas and cusolver targets are incorrect. This module must be called
 from the same CMakeLists.txt as the first `find_project(CUDAToolkit)` to
 patch the targets.
 
+For all version of Cmake the dependencies of cusparse are incorrect.
```

Comment:
```suggestion
For all versions of CMake, the dependencies of cusparse are incorrect.
```

Reply:
Is there an upstream fix needed to CMake? If an issue has been reported upstream, can it be cross-referenced here? "All versions are broken" sounds like a statement that will hopefully be changed in the future.

Reply:
I am in the process of figuring out if this line of docs is correct or not. Looking at the history it is unclear to me if this is needed for a subset of CUDA versions, or if this was done due to incorrect downstream CUDA linking and we 'fixed' it in rapids-cmake.

Reply:
I can't find an occurence where cusparse depends on cublas. Playing around with link lines using the static libraries I can't find any configuration order that causes a breakage ( 11.0, 11.4, 11.6. 11.8 ).

So I will drop the cusparse -> cublas linkage.